### DISCUSSION TITLE
warn when current build and equipment does not match templates

### DISCUSSION THREAD

**Discussion by Taschenbuch** [blish-hud/Blish-HUD#938] [Module Ideas]
I would like a module called "Legendary child locks". You get a list of your equipment and build tabs, and when you select the "Locked" state for each the addon pulls & records the status of that tab. Then, when it detects that configuration has changed, it puts a little danger badge on its menu icon in the top left blish icon list if you are currently active on that build/eq.     For example "I'll just put relic of speed on for this event, I'll remember to change it back..". I uh can't explain how many instances of finding out I was running with a broken build for some serious period of time :/ 
https://discord.com/channels/531175899588984842/534490472224391169/1202357089222283285

similiar but not same idea:
"Had a random idea during my stream: A build snapshot/changelog add-on.
Add-on starts and queries the API for the current characters build.
Every X minutes it requests the API again. If any changes, these are logged.
This would be helpful for when changing traits or stats while testing builds or if you accidentally change a legendary weapon or something."
https://discord.com/channels/531175899588984842/534490472224391169/1243859683551219723

**Comment by gartkb** [comment]
Thinking this through slightly more, I think when you are being warned that the actual equipment/build does not match the locked reference build/eq the danger badge should have a hover that details what is different.
